client/src/main/scala/io/delta/sharing/client/DeltaSharingClient.scala

```diff
@@ -88,6 +87,16 @@ private[sharing] trait PaginationResponse {
   def nextPageToken: Option[String]
 }
 
+private[sharing] trait PaginationRequest {
```

Comment:
Why extracting this trait out?

Reply:
this is to make GetQueryInfo api to be able to support the pagination when retrieving the query result. so extract some common properties.